core/src/avm2/globals/xml_list.rs

```diff
@@ -232,3 +232,13 @@ pub fn text<'gc>(
     }
     Ok(XmlListObject::new(activation, nodes, Some(xml_list.into())).into())
 }
+
+pub fn copy<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let list = this.unwrap().as_xml_list_object().unwrap();
+    let children = (*list.children()).clone();
```

Comment:
I don't think this is correct - the [docs](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/XMLList.html#copy()) say `The copy is a duplicate of the entire tree of nodes.`, but this only clones the first layer.